core/include/webview/detail/engine_base.hh

```diff
@@ -318,6 +319,19 @@ protected:
     }
   }
 
+  void dispatch_size_default(bool m_owns_window) {
+    if (!m_owns_window) {
+      return;
+    };
+    dispatch([this]() {
+      if (!m_is_size_set) {
```

Comment:
Note: On Windows, this lambda is executed 10 times when running the basic C++ example without calling `set_size()`, and 4 times with `set_size()`.